Sources/Mailgun/Mailgun.swift

```diff
@@ -111,10 +148,11 @@ public struct Mailgun: MailgunProvider {
     ///
     /// - Parameters:
     ///   - content: Message
+    ///   - domain: String? 
     ///   - container: Container
     /// - Returns: Future<Response>
-    public func send(_ content: Message, on container: Container) throws -> Future<Response> {
-        return try postRequest(content, endpoint: "messages", on: container)
+    public func send(_ content: Message, domain: String? = nil, on container: Container) throws -> Future<Response> {
```

Comment:
It might be better to pass in a `DomainConfig` instead given my other concerns. A default `DomainConfig` could be set within `Mailgun` instead and that could be used in the case that no `DomainConfig` is passed in.

If we suggest that users set up static vars for their domains like
```swift
extension DomainConfig {
    static var euDomain: DomainConfig {
        return Mailgun.DomainConfig("mg.example.com", region: .eu)
    }
}
```

Then `send` could be called like 
```swift
send(_ content: ..., domain: .euDomain, on container: ...)
```

Reply:
I really like the statis vars, but as its not possible to create the domain twice i guess it's enough to work with strings?